src/panels/config/devices/ha-config-devices-dashboard.ts

```diff
@@ -378,21 +377,24 @@ export class HaConfigDeviceDashboard extends LitElement {
             batteryEntityPair && batteryEntityPair[1]
               ? this.hass.states[batteryEntityPair[1]]
               : undefined;
-          const batteryIsBinary =
-            battery && computeStateDomain(battery) === "binary_sensor";
+          const batteryDomain = battery ? computeStateDomain(battery) : "";
 
-          return battery && (batteryIsBinary || !isNaN(battery.state as any))
+          return battery &&
+            (batteryDomain === "sensor" ||
+              batteryDomain === "binary_sensor" ||
+              !isNaN(battery.state as any))
```

Comment:
We don't do anything with these entities, so I think it should be removed here?

Reply:
On this page, the battery is showed near the end of the row in the table where a battery entity exists on a device.
The battery entity is used to show the state and if no state is present (because entity is a binary sensor) then the state is sort of reflected in the battery icon to indicate if it's charging or not as well to show if the battery is fully charged or empty.